chainer_chemistry/models/prediction/base.py

```diff
@@ -134,6 +134,38 @@ def _forward(self, data, fn, batchsize=16,
     def save_pickle(self, filepath, protocol=None):
         """Save the model to `filepath` as a pickle file
 
+        This function send the parameters to CPU before saving the model so
+        that the pickled file can be loaded with in CPU-only environment. 
+
+        Saved pickle file can be loaded with `load_pickle` static method.
+
+        Note that the transportability of the saved file follows the
+        specification of `pickle` module, namely serialized data depends on the
+        specific class or attribute structure when saved. The file may not be
+        loaded in different environment (python version is different or
```

Comment:
I thinks description in the parenthesis is a bit redundant. How about `(versions of Python or dependent libraries)`?